dashboard/config/foorm/forms/surveys/pd/ayw_workshop_pre_survey.0.json

```diff
@@ -0,0 +1,1371 @@
+{
+ "pages": [
+  {
+   "name": "page 1",
+   "elements": [
+    {
+     "type": "html",
+     "name": "question2",
+     "html": "<h2>{workshop_course} Workshop Pre-Survey</h2>"
+    },
+    {
+     "type": "html",
+     "name": "612_facilitator_post_header",
+     "html": "<h2></h2>"
+    },
+    {
+     "type": "radiogroup",
+     "name": "workshop_course",
+     "title": "For which Code.org course are you attending a workshop?",
+     "choices": [
+      {
+       "value": "CS Principles",
+       "text": "CS Principles"
+      },
+      {
+       "value": "CS Discoveries",
+       "text": "CS Discoveries"
+      }
+     ]
+    },
+    {
+     "type": "radiogroup",
+     "name": "is_virtual",
+     "title": "Is the workshop you're attending in-person, or virtual?",
+     "choices": [
+      {
+       "value": "false",
+       "text": "In-Person"
+      },
+      {
+       "value": "true",
+       "text": "Virtual"
+      }
+     ]
+    },
+    {
+     "type": "radiogroup",
+     "name": "question4",
+     "title": " Are you planning to attend this upcoming workshop?",
+     "choices": [
+      {
+       "value": "yes",
+       "text": "Yes"
+      },
+      {
+       "value": "no",
+       "text": "No"
+      },
+      {
+       "value": "unsure",
+       "text": "Unsure / Don't Know Yet"
+      }
+     ]
+    },
+    {
+     "type": "panel",
+     "name": "csp_most_recent_unit_panel",
+     "elements": [
+      {
+       "type": "dropdown",
+       "name": "csp_unit",
+       "visibleIf": "{workshop_course} = 'CS Principles'",
+       "title": "In which unit from CS Principles is the lesson you most recently taught?",
+       "choices": [
+        {
+         "value": "Unit_1",
```

Comment:
nit: make the values all lowercase for consistency (ex unit_1)

Reply:
Fixed EXCEPT FOR ONE SPOT.  But thats for workshop_course radio item -- which I added in the event we wouldn't have the variable present in this survey, but it sounds like we will.  yes?  If so we can get rid of question name: workshop_course

Reply:
yes we will have workshop_course and is_virtual

Reply:
woot.  removing those two items the.